src/watertap_contrib/reflo/solar_models/surrogate/trough/test_trough_surrogate.py

```diff
@@ -46,6 +47,8 @@
     unscaled_variables_generator,
 )
 
+random.seed(42)
```

Comment:
Really excited to see that you got this approach to work! One question here--how does setting this random seed get applied in the test code below?

Reply:
It was to catch any other code that calls random without setting a seed. I'm not sure if there are such calls being made, so I've removed this line to see if the tests still pass